starter-code/src/array-functions.js

```diff
@@ -1,33 +1,64 @@
-var ArrayFunctions = function (){}
+class ArrayFunctions {
+  // 1) Define a function that takes an array with numbers and prints all the elements of the array, separated by " --- "
+  // ArrayFunctions.printSpecial([12, 33, 144, 122])
+  // 12 -- 33 -- 144 -- 122
 
-// 1) Define a function that takes an array with numbers and prints all the elements of the array, separated by " --- "
-// ArrayFunctions.printSpecial([12, 33, 144, 122])
-// 12 -- 33 -- 144 -- 122
+  static printSpecial(array) {
```

Comment:
Using static methods was mandatory in this part of our homework. As I understood, these kind of methods are called only on the class itself and can't be called on the instances of the class. I read more about it on this page: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static